services/worker/tests/test_job_manager.py

```diff
@@ -115,6 +117,8 @@ def test_backfill(priority: Priority, app_config: AppConfig) -> None:
     assert job_result["is_success"]
     assert job_result["output"] is not None
     assert job_result["output"]["content"] == {"key": "value"}
+    assert job_result["duration"] is not None
```

Comment:
this is the only test that checks that duration is set as expected for successful jobs. should I test it anywhere else?